### ISSUE TITLE
[4.0] Backend template breaks bootstrap 4 btn-outline-* button classes by forcing zero border-width and text color

### ISSUE THREAD

**Issue by ggppdk** [joomla/joomla-cms#21302]
### Steps to reproduce the issue

Try to use these in the backend

```html
<button type="button" class="btn btn-outline-primary">Primary</button>
<button type="button" class="btn btn-outline-secondary">Secondary</button>
<button type="button" class="btn btn-outline-success">Success</button>
<button type="button" class="btn btn-outline-danger">Danger</button>
<button type="button" class="btn btn-outline-warning">Warning</button>
<button type="button" class="btn btn-outline-info">Info</button>
<button type="button" class="btn btn-outline-light">Light</button>
<button type="button" class="btn btn-outline-dark">Dark</button>
```

### Expected result
You get result similar to
https://getbootstrap.com/docs/4.0/components/buttons/#outline-buttons

### Actual result
- No border is shown !
- Text color is hard-coded to white

### System information (as much as possible)
4.0-dev

### Additional comments
If we are going to add zero border-width to .btn class
then we should exclude classes .btn-outline-*
or
find do some other better alternative

**Comment by ciar4n** [comment]
To my knowledge, bootstrap does not allow us to disable btn-outlines-*.

**Comment by ciar4n** [comment]
PR created.. #21322

**Comment by ghost** [comment]
closed as having Pull Request #21322
